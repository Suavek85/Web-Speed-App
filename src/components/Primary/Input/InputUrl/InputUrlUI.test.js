import React from 'react'
import { shallow } from 'enzyme'
import InputButtonUrlUI from './InputUrlUI'

describe('InputButtonUrlUI snapshot', () => {

  it('should match the snapshot', () => {
    const wrapper = shallow(<InputButtonUrlUI />)
    expect(wrapper).toMatchSnapshot()
  })

})