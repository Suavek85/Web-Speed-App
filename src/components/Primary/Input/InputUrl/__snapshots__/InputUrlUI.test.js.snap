// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputButtonUrlUI snapshot should match the snapshot 1`] = `
<label
  htmlFor="urlinput"
>
  <input
    autoComplete="off"
    autoFocus={true}
    id="urlinput"
    placeholder=" "
    type="url"
  />
  <span>
    Enter your website url in this format https://www.bbc.co.uk/
  </span>
  <span />
</label>
`;
